tox.ini

```diff
@@ -53,9 +52,6 @@ extras =
     alldeps: all
 
 commands =
-    # FIXME: Not sure why need this for tox to see the C-ext
-    python setup.py build_ext --inplace
-    # End of FIXME
     pip freeze
     !cov: pytest --pyargs synphot {toxinidir}/docs {posargs}
```

Comment:
@jdavies-st , I do have `--pyargs` here. Am I using it wrong?

Reply:
With `changedir` commented out above, because you are running `pytest --pyargs` from within a directory that has `synphot` as a subdir, every `import synphot` will import it from the local checkout, not the installed location, following normal python import rules that always imports the local dir thing before checking python path.